package haptix.comm.msgs;

message hxJoint
{
  required string name          = 1;
  required float pos            = 2;
  required float vel            = 3;
  required float acc            = 4;
  required float torque_motor   = 5;
  required float torque_passive = 6;
}
